src/styles/Login.css.js

```diff
@@ -1,5 +1,10 @@
 import { css } from 'react-emotion';
 
+const colorBlue = '#2196F3';
+const colorWhite = '#fff'
+const inputBackgroundColor = 'rgba(238, 238, 238, .6)';
+const inputBorderColor = 'rgba(0, 0, 0, .38)';

```

Comment:
Thanks for pulling out the colors into variables. I think the next step would be to create a colors.js file that we could import into more `*.css.js` files down the line as we refactor.